steam.py

```diff
@@ -189,11 +190,22 @@ def get_game_sales():
         if wishlist_json[app_id]['sale_history']['sale_start'] != None:
             games_on_sale[app_id] = wishlist_json[app_id]
     return games_on_sale
-
+ 
 def build_empty_sale_history():
     sale_history = {'sale_start':None, 'last_sale_start':None, 'last_sale_end':None}
     return sale_history
 
+# Doesnt do anything until it is called in Noah's friday loop.
+# randomly fetches a tuple from Friday/phrases and returns the 2nd value from the tuple (the string we want.)
+def friday_phrase_randomizer():
+    with open(friday_file) as f:

```

Comment:
This is a good syntax to use, it automatically manages the file closing, we should update all of our file access to use this, just something to keep in mind, nothing to do now.